tsconfig.json

```diff
@@ -9,6 +9,7 @@
     "module": "ESNext",
     "moduleResolution": "Node",
     "useDefineForClassFields": false,
+    "allowSyntheticDefaultImports": true,
```

Comment:
why is it needed?

Reply:
```
Module 'dockit-core/node_modules/copy-to-clipboard/index' can only be default-imported using the 'allowSyntheticDefaultImports' flag
```
I haven't found other workarounds.

Reply:
can you please check if this doesn't break our build of dockit-core
you can use [CLI's `link` command](https://backlight.dev/docs/cli#link) to see the built package